### PR TITLE
Playwright: Editor Tracking -- Add more block event tests

### PR DESCRIPTION
**Pull request by dpasque**
#### Changes proposed in this Pull Request

From this test plan map: pciE2j-QC-p2

... This implements the remainder of the tests under `editor-tracking__block-events.ts`, with a few changes...

1. Focusing the moved and deletion events just on site editor template parts seemed too narrowly focused in the grand scheme of things. (This was a copy over from the old scripts). I've now tackled just basic use cases for those events.
2. The original block-events script was getting too long. `wpcom_block_inserted` is by far our most extensively tested event (with good reason), so I've given it its own script, and moved the other block-based events to their own file. This keeps things speedy with parallelization!

#### Testing instructions

- [x] Gutenberg desktop passes
- [x] Gutenberg mobile passes

Related to #

### CODE REVIEW

File: test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+						{
+							matchingProperties: { from_template_selector: true },
+						}
+					);
+					expect( eventDidFire ).toBe( true );
+				} );
+			} );
+		} );
+
+		describe( 'In the site editor', function () {
```
Comment (dpasque) [inline test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts:175]: The rename throws off the diff here a bit. Other than some re-organization, here below is the only really meaningful addition to this script.

File: packages/calypso-e2e/src/lib/pages/full-site-editor-page.ts [hunk 2]
```diff
@@ -210,6 +243,26 @@ export class FullSiteEditorPage {
...
+			`${ selectors.focusedBlock( blockEditorSelector ) },${ selectors.parentOfFocusedBlock(
+				blockEditorSelector
+			) }`
+		);
+		await addedBlockLocator.waitFor();
+
+		return await getIdFromBlock( addedBlockLocator );
+	}
+
+	// TODO: Add this to the Gutenberg component to make it re-usable across editors!
```
Comment (dpasque) [inline packages/calypso-e2e/src/lib/pages/full-site-editor-page.ts:265]: These TODOs are just ideas I had while I was in here, so adding them for future me :)

File: test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts (hunk 1 above)
Comment (worldomonation) [reply test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts:175]: I assume we talked about this at some point, but this is a long test file. Have you found this to be reliable?

**Review by worldomonation — APPROVED** [review]
I'm flying Thursday and taking Friday off to recover, so in order to not block your progress I'll be approving the PR.

File: test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts (hunk 1 above)
Comment (dpasque) [reply test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts:175]: Yep, so far! The length is slightly misleading as it's really three distinct test flows in one file. Any one flow isn't too long and they've been reliable.

I've been trying to strike a balance between keeping the files lean, but also not making too many (e.g. I think that one file per event might grow a bit too much). How do you think that balance has been working so far? I'm very open to breaking this out into smaller files!

Let me know what you think @worldomonation and I can do that in a future pr! 😄

**Pull request merged**